### PR TITLE
Feature/#122 rescale window

### PR DESCRIPTION
**Pull request by PhlppKrmr**
Sorry in advance for the humongous PR ;)
Works on Chromium and Firefox, @Mac guys, I'd very much appreciate it if you'd check if it works on Safari as well.

### CODE REVIEW

**Comment by PhlppKrmr** [comment]
One more comment, I removed the Log for this version, if we want it back I'll add it again.

**Review by nicolaskolbenschlag — APPROVED** [review]
Looks good!

**Comment by leandertolksdorf** [comment]
Awesome! I think the background could need some kind of color shades or some separation of control units but that's something for another ticket.

**Review by leandertolksdorf — APPROVED** [review]
Also works in Safari

File: Apps/frontend/src/views/Layout.svelte [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    return socket;
+  };
+
+  /**
+   * On receiving a socket message, update buffer of waves.
+   *
+   * @param {MessageEvent} messageEvent - has poperty (Float64Array) data
+   */
+  const socketOnMessage = (messageEvent) => {
```
Comment (leon-juenemann) [inline Apps/frontend/src/views/Layout.svelte:52]: Feels wrong to have the websocket and update buffer logic in a file named Layout. Not a big deal as it wasn't much better before. (originally Oscilloscope.svelte)

**Review by leon-juenemann — COMMENTED** [review]
I really like the design. Looks like quite a bit of work. Kind of hard to review because it is so large. Tested the UI and feels good!

File: Apps/frontend/src/views/Layout.svelte (hunk 1 above)
Comment (PhlppKrmr) [reply Apps/frontend/src/views/Layout.svelte:52]: Git didn't pick up on it cause it wasn't ;)
I created `Layout.svelte` separately to build the whole Layout from scratch so I don't have to mingle with the existing one. Will rename to `Oscilloscope.svelte`, was a good point.

**Review by jandegen — APPROVED** [review]

**Pull request merged**